scripts/deploy-backend.sh

```diff
@@ -100,29 +100,30 @@ update_kibana()
   cd tdrs-backend || exit
 
   # Run template evaluation on manifest
-  yq eval -i ".applications[0].services[0] = \"es-${backend_app_name}\""  manifest.proxy.yml
+  # 2814: need to update this and set it to env instaead of app name
+  yq eval -i ".applications[0].services[0] = \"es-${env}\""  manifest.proxy.yml
   yq eval -i ".applications[0].env.CGAPPNAME_PROXY = \"${CGAPPNAME_PROXY}\""  manifest.kibana.yml
 
   if [ "$1" = "rolling" ] ; then
-        # Do a zero downtime deploy.  This requires enough memory for
-        # two apps to exist in the org/space at one time.
-        cf push "$CGAPPNAME_PROXY" --no-route -f manifest.proxy.yml -t 180 --strategy rolling || exit 1
-        cf push "$CGAPPNAME_KIBANA" --no-route -f manifest.kibana.yml -t 180 --strategy rolling || exit 1
-    else
-        cf push "$CGAPPNAME_PROXY" --no-route -f manifest.proxy.yml -t 180
-        cf push "$CGAPPNAME_KIBANA" --no-route -f manifest.kibana.yml -t 180
-    fi
-    
-    cf map-route "$CGAPPNAME_PROXY" apps.internal --hostname "$CGAPPNAME_PROXY"
-    cf map-route "$CGAPPNAME_KIBANA" apps.internal --hostname "$CGAPPNAME_KIBANA"
-
-    # Add network policy allowing Kibana to talk to the proxy and to allow the backend to talk to Kibana
-    cf add-network-policy "$CGAPPNAME_KIBANA" "$CGAPPNAME_PROXY" --protocol tcp --port 8080
-    cf add-network-policy "$CGAPPNAME_BACKEND" "$CGAPPNAME_KIBANA" --protocol tcp --port 5601
-    cf add-network-policy "$CGAPPNAME_FRONTEND" "$CGAPPNAME_KIBANA" --protocol tcp --port 5601
-    cf add-network-policy "$CGAPPNAME_KIBANA" "$CGAPPNAME_FRONTEND" --protocol tcp --port 80
-
-    cd ..
+      # Do a zero downtime deploy.  This requires enough memory for
+      # two apps to exist in the org/space at one time.
+      cf push "$CGAPPNAME_PROXY" --no-route -f manifest.proxy.yml -t 180 --strategy rolling || exit 1
+      cf push "$CGAPPNAME_KIBANA" --no-route -f manifest.kibana.yml -t 180 --strategy rolling || exit 1
+  else
+      cf push "$CGAPPNAME_PROXY" --no-route -f manifest.proxy.yml -t 180
+      cf push "$CGAPPNAME_KIBANA" --no-route -f manifest.kibana.yml -t 180
+  fi
+  
+  cf map-route "$CGAPPNAME_PROXY" apps.internal --hostname "$CGAPPNAME_PROXY"
+  cf map-route "$CGAPPNAME_KIBANA" apps.internal --hostname "$CGAPPNAME_KIBANA"
+
+  # Add network policy allowing Kibana to talk to the proxy and to allow the backend to talk to Kibana
+  cf add-network-policy "$CGAPPNAME_KIBANA" "$CGAPPNAME_PROXY" --protocol tcp --port 8080
+  cf add-network-policy "$CGAPPNAME_BACKEND" "$CGAPPNAME_KIBANA" --protocol tcp --port 5601
+  cf add-network-policy "$CGAPPNAME_FRONTEND" "$CGAPPNAME_KIBANA" --protocol tcp --port 5601
+  cf add-network-policy "$CGAPPNAME_KIBANA" "$CGAPPNAME_FRONTEND" --protocol tcp --port 80
+
+  cd ..
```

Comment:
these changes are solely linting

Reply:
Thank you for calling that out haha.

Reply:
I think all the kibana and proxy deployment code needs to be moved under the if statement that checks if this is an `initial` deployment since we have one instance of each now.

Reply:
since the elastic and kibana are per environment now, I think we should move their deployment to separate file, then keep the map-route and add-network-policy in for all deployments.